plugins/source/aws/resources/services/rds/engine_versions.go

```diff
@@ -32,25 +32,9 @@ func EngineVersions() *schema.Table {
 				},
 			},
 			{
-				Name:     "engine",
+				Name:     "_engine_version_hash",
```

Comment:
Don't love the name of this column... open to suggestions!

Reply:
i think it's fine for now